features/marker.yml

```
@@ -0,0 +1,8 @@
+name: ::marker
+description: The `::marker` CSS pseudo-element selects the font, text, and color of a list item marker.
```

"selects" feels like the wrong verb here. CSS selectors select _elements_.
Maybe "styles".

---

Sorry this is my fault, I had an incomplete thought on this in https://github.com/web-platform-dx/web-features/pull/1764#discussion_r1777341651. We probably want to say something closer to this:

```suggestion
description: The `::marker` CSS pseudo-element selects a list item marker for styling numbers or bullets.
```

---

Good timing! I just opened up a [PR](https://github.com/web-platform-dx/web-features/pull/1869) with more pseudo elements.